### DISCUSSION TITLE
Hotkey to close Google Assistant Window

### DISCUSSION THREAD

**Discussion by LeonitusVIII** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#380] [Ideas]
Loving this program so much! One thing that is bugging me, which hopefully is an easy idea, is to have a Hotkey to close the assistant box. Not to exit the program but to put it back in the tray and not show on the screen or taskbar anymore. 

Otherwise, it's working really well! Thanks for all the hard work.

**Comment by sumeshir26** [comment]
@LeonitusVIII There should be a setting for that called "Hotkey Behaviour". You can set it to Launch/Close App.

**Reply by LeonitusVIII** [reply]
Ahh, I see that is a dropdown, but I think it would still be good to have a separate button OR have a 3-way toggle (not sure what else to call this) so that toggle microphone can still be performed as well as closing the app. Otherwise, you are choosing between toggling the microphone and closing the app with that button.
